addons/mail/static/src/core/common/message.js

```diff
@@ -416,16 +412,6 @@ export class Message extends Component {
         });
     }
 
-    onTouchstart() {
```

Comment:
Those are still defined in template 

t-on-touchstart.capture="onTouchstart"
                t-on-touchend.capture="onTouchend"